docs-src/references/strongly-typed-errors/cause-bad-update.md

```diff
@@ -0,0 +1,15 @@
+---
+id: cause-bad-update
+title: Cause Bad Update
+description: Explanation for Cause Bad Update error message, and how to fix it.
+sidebar_label: Cause Bad Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+This error indicates that the [Workflow Execution](/concepts/what-is-a-workflow-execution) update could not be completed.
```

Comment:
```suggestion
This error indicates that a [Workflow Execution](/concepts/what-is-a-workflow-execution) update could not be completed.
```

Also need a TODO to link to the Workflow Update concept page once that's available.